internal/service/databaseuser/resource_database_user.go

```diff
@@ -43,7 +40,7 @@ func Resource() resource.Resource {
 	}
 }
 
-type tfDatabaseUserModel struct {
```

Comment:
I think we should use RequiresReplace also for Roles and Labels, for the same reason we did for Scopes, until the SDK is changed.

Reply:
I didn't include it to Roles and Labels because in the old sdk it was not like this it has omitempty. I understand it makes sense to do it, but I think that is better not to change the behaviour just now, at least until we have a more permanent solution for this cases